packages/trpc/server/routers/viewer.tsx

```diff
@@ -1437,6 +1437,18 @@ const loggedInViewerRouter = createProtectedRouter()
         },
       });
     },
+  })
+  .query("bookingUnconfirmedCount", {
+    async resolve({ ctx }) {
+      const { prisma, user } = ctx;
+      return await prisma.booking.count({
+        where: {
```

Comment:
Should we group recurring events here or somewhere else? Having 3 regular bookings unconfirmed and 12 others as part of a recurring event should perhaps show 4 bookings unconfirmed, not 15.

Reply:
probably a good idea @emrysal

Reply:
Just pushed an expansion of the tRPC query to support it.